### DISCUSSION TITLE
Download Avalon Layout for PRIMENG

### DISCUSSION THREAD

**Discussion by kaddamienOutlook** [primefaces/community#1552] [PrimeNG Templates]
Hi
I purshased 'Avalon Layout for PRIMENG' but the seller didn't give me the link to download it. I wrote many times but no response.
can somebody here help me to download it ?

**Comment by cetincakiroglu** [comment]
Hi,

We are sorry for the delayed response, did you email the contact@primetek.com.tr ?
